plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -8035,6 +8040,27 @@ protected Session withoutSmallFileThreshold(Session session)
                 .build();
     }
 
+    @Test
+    protected void testBucketTransformationsWithVarbinary()
```

Comment:
Can this be added to existing `testBucketTransform` method which covers bucketing for other types?